src/statd/python/cli_pretty/cli_pretty.py

```diff
@@ -1272,27 +1311,29 @@ def main():
 
     parser.add_argument('-t', '--test', action='store_true', help='Enable unit test mode')
 
-    parser_show_routing_table = subparsers.add_parser('show-routing-table', help='Show the routing table')
-    parser_show_routing_table.add_argument('-i', '--ip', required=True, help='IPv4 or IPv6 address')
+    subparsers.add_parser('show-routing-table', help='Show the routing table') \
+        .add_argument('-i', '--ip', required=True, help='IPv4 or IPv6 address')
+
+    subparsers.add_parser('show-interfaces', help='Show interfaces') \
+        .add_argument('-n', '--name', help='Interface name')
+    
+    subparsers.add_parser('show-software', help='Show software versions') \
+        .add_argument('-n', '--name', help='Slotname')
 
-    parser_show_interfaces = subparsers.add_parser('show-interfaces', help='Show interfaces')
-    parser_show_interfaces.add_argument('-n', '--name', help='Interface name')
+    subparsers.add_parser('show-dhcp-server', help='Show DHCP server') \
+        .add_argument("-s", "--stats", action="store_true", help="Show server statistics")
 
-    parser_show_bridge_mdb = subparsers.add_parser('show-bridge-mdb', help='Show bridge MDB')
-    parser_show_bridge_stp = subparsers.add_parser('show-bridge-stp',
-                                                   help='Show spanning tree state')
+    subparsers.add_parser('show-bridge-mdb', help='Show bridge MDB')
 
-    parser_show_software = subparsers.add_parser('show-software', help='Show software versions')
-    parser_show_software.add_argument('-n', '--name', help='Slotname')
+    subparsers.add_parser('show-bridge-stp', help='Show spanning tree state')
 
-    parser_show_hardware = subparsers.add_parser('show-hardware', help='Show USB ports')
+    subparsers.add_parser('show-hardware', help='Show USB ports')
 
-    parser_show_ntp_sources = subparsers.add_parser('show-ntp', help='Show NTP sources')
+    subparsers.add_parser('show-ntp', help='Show NTP sources')
 
-    parser_show_boot_order = subparsers.add_parser('show-boot-order', help='Show NTP sources')
+    subparsers.add_parser('show-boot-order', help='Show NTP sources')
```

Comment:
Not sure that these changes belong in this commit.  If you want to sort (alphabetically) and/or clean up the parser setup, please put that in a separate commit, or just revert the above change.